src/js/components/SelectMultiple/SelectMultipleContainer.js

```diff
@@ -0,0 +1,595 @@
+import React, {
+  forwardRef,
+  useCallback,
+  useContext,
+  useEffect,
+  useRef,
+  useState,
+} from 'react';
+import { ThemeContext } from 'styled-components';
+import { FormUp } from 'grommet-icons/icons/FormUp';
+
+import { setFocusWithoutScroll } from '../../utils';
+
+import { defaultProps } from '../../default-props';
+
+import { Box } from '../Box';
+import { Button } from '../Button';
+import { CheckBox } from '../CheckBox';
+import { InfiniteScroll } from '../InfiniteScroll';
+import { Keyboard } from '../Keyboard';
+import { Text } from '../Text';
+import { TextInput } from '../TextInput';
+import { SelectionSummary } from './SelectionSummary';
+import {
+  StyledContainer,
+  OptionsContainer,
+  SelectOption,
+} from '../Select/StyledSelect';
+import {
+  applyKey,
+  getOptionLabel,
+  getOptionValue,
+  checkDisabled,
+} from '../Select/utils';
+import { EmptySearchOption } from '../Select/EmptySearchOption';
+
+const SelectMultipleContainer = forwardRef(
+  (
+    {
+      allOptions,
+      children = null,
+      disabled: disabledProp,
+      disabledKey,
+      dropHeight,
+      emptySearchMessage = 'No matches found',
+      help,
+      id,
+      labelKey,
+      limit,
+      onChange,
+      onClose,
+      onKeyDown,
+      onMore,
+      onSearch,
+      optionIndexesInValue,
+      options,
+      replace = true,
+      searchPlaceholder,
+      search,
+      setSearch,
+      usingKeyboard,
+      value = [],
+      valueKey,
+      showSelectedInline,
+    },
+    ref,
+  ) => {
+    const theme = useContext(ThemeContext) || defaultProps.theme;
+    const [activeIndex, setActiveIndex] = useState(-1);
+    const [keyboardNavigation, setKeyboardNavigation] = useState(usingKeyboard);
+    const searchRef = useRef();
+    const optionsRef = useRef();
+    const [disabled, setDisabled] = useState(disabledProp);
+    const activeRef = useRef();
+    const [showA11yLimit, setShowA11yLimit] = useState();
+    const clearRef = useRef();
+
+    // for keyboard/screenreader, keep the active option in focus
+    useEffect(() => {
+      if (activeIndex) activeRef.current?.focus();
+    }, [activeIndex]);
+
+    // set initial focus
+    useEffect(() => {
+      // need to wait for Drop to be ready
+      const timer = setTimeout(() => {
+        const clearButton = clearRef.current;
+        if (clearButton && clearButton.focus) {
+          setFocusWithoutScroll(clearButton);
+        } else if (searchRef && searchRef.current) {
+          const searchInput = searchRef.current;
+          if (searchInput && searchInput.focus) {
+            setFocusWithoutScroll(searchInput);
+          }
+        } else if (activeRef.current) {
+          setFocusWithoutScroll(activeRef.current);
+        } else if (optionsRef.current) {
+          setActiveIndex(0);
+        }
+      }, 100);
```

Comment:
Can you add a comment indicating why "100" was chosen?

Reply:
Yes, done